src/Http/Controllers/Admin/ModuleController.php

```diff
@@ -250,6 +252,13 @@ abstract class ModuleController extends Controller
      */
     protected $fieldsPermissions = [];
 
+    /**
+     * Determines if draft revisions can be added on top of published content.
+     *
+     * @var bool
+     */
+    protected $enableDraftRevisions = false;
```

Comment:
Can we add a todo to make a setter for this in 3.x?

Reply:
Yes, feel free to add any anotations you would find helpful.